ecosystem/sep-0010.md

```diff
@@ -106,7 +108,9 @@ Request Parameters:
 
 Name      | Type          | Description
 ----------|---------------|------------
-`account` | `G...` string | The stellar account that the wallet wishes to authenticate with the **Server** 
+`account` | `G...` string | The stellar account that the **Client** wishes to authenticate with the **Server** 
+`memo` | string | (optional) The memo to attach to the challenge transaction, for `hash` this should be base64-encoded. See the [Memo](#memos) section for details.
+`memo_type` | string | (optional) The type of memo to attach to transaction, one of `text`, `id` or `hash`.
```

Comment:
❓ Could we start with supporting only a single memo type, ideally ID, since then it matches muxed accounts. See my other comment about this further up.